YoutubePlaylistDownloader/Languages/中文.xaml

```diff
@@ -3,163 +3,180 @@
+    <s:String x:Key="Remove">删除</s:String>
+    <s:String x:Key="Id">ID：</s:String>
     <s:String x:Key="Save">保存</s:String>
-    <s:String x:Key="PreferFPS">喜欢的视频有最高FPS</s:String>
-    <s:String x:Key="AudioOnly">下载最好的音频和转变 没有格式化</s:String>
-    <s:String x:Key="StillDownloadingSubscriptionsExit">这个算机程序正在下载您的视频. 单击是会关上电脑程序. 你确定你要退出吗?</s:String>
-    <s:String x:Key="LastVideoDownloadDate">最近更新时间: </s:String>
-    <s:String x:Key="Help">求助</s:String>
-    <s:String x:Key="SubscriptionsHelp">订阅是频道电脑程序下载自动. 例如用户添加频道9月1日. 视频上传9月2日自动下载. 以前的视频没有下载.删除的项目不会再次下载. 订阅是定制. 单击订阅然后单击配置.
+    <s:String x:Key="PreferFPS">更想要下载最高FPS的视频</s:String>
+    <s:String x:Key="AudioOnly">下载纯净的音频，不转换为任意格式</s:String>
+    <s:String x:Key="StillDownloadingSubscriptionsExit">程序正在下载你订阅的视频，点击是将会关闭程序且那些视频将不会被下载直到你重新打开程序。
+你确定要退出吗？</s:String>
+    <s:String x:Key="LastVideoDownloadDate">最后一次更新：</s:String>
+    <s:String x:Key="Help">帮助</s:String>
+    <s:String x:Key="SubscriptionsHelp">订阅频道意味着程序会在订阅被添加后自动下载任何新出的视频
+        
+例如，如果你在九月一日添加了频道"x"的订阅且频道"x"在九月二号发布了新视频，它就会被自动下载。
+        
+以前的视频将不会被下载，且如果你删除了一个特定的视频它将不会被再次下载。 
+        
+所有的订阅均可自定义且带有独自的设置，要访问订阅设置从主菜单点击订阅然后点击你想要设置的频道旁的设置即可。
+
+请注意如果程序未运行，新视频就不会被下载。
     </s:String>
-    <s:String x:Key="ContactTheDeveloper">联络我</s:String>
-    <s:String x:Key="ContactTheDeveloperHelp">有联的方法络我 (shaked6540):首先是 reddit. 我的 reddit 是 https://www.reddit.com/user/shaked6540/. 请发给我 PM 我会尽快回复您.
-我的 github 是 https://github.com/shaked6540/YoutubePlaylistDownloader/issues</s:String>
-    <s:String x:Key="AutomaticallyCheckForUpdates">自动检查更新</s:String>
-    <s:String x:Key="WhatsNew">新功能</s:String>
-    <s:String x:Key="SubscriptionsUpdateDelay">更新我的订阅 </s:String>
-    <s:String x:Key="Minutes">分钟</s:String>
-    <s:String x:Key="SaveDownloadOptions">保存我的下载配置</s:String>
-    <s:String x:Key="DownloadSpeed">下载速度 </s:String>
-    <s:String x:Key="CreditsAndContributors">Many thanks to these people for translating this software: 
+    <s:String x:Key="ContactTheDeveloper">联系我</s:String>
+    <s:String x:Key="ContactTheDeveloperHelp">有几种联系我 (shaked6540) 的方法： 
+
+首先就是通过Reddit，在Reddit上给我发送消息后我就会尽快回复你，这是我的个人资料链接：https://www.reddit.com/user/shaked6540/
+
+其次就是通过GitHub，开启一个新的issue后我就会尽快回复你，这是链接：https://github.com/shaked6540/YoutubePlaylistDownloader/issues</s:String>
+    <s:String x:Key="AutomaticallyCheckForUpdates">在程序启动时自动检测更新</s:String>
+    <s:String x:Key="WhatsNew">更新日志</s:String>
+    <s:String x:Key="SubscriptionsUpdateDelay">每</s:String>
+    <s:String x:Key="Minutes">分钟后更新订阅</s:String>
+    <s:String x:Key="SaveDownloadOptions">保存我的下载设置</s:String>
+    <s:String x:Key="DownloadSpeed">下载速度：</s:String>
+    <s:String x:Key="Changelog">更新日志</s:String>
+    <s:String x:Key="DongChunrui">Dong Cunrui (董存瑞)</s:String>
+    <s:String x:Key="CreditsAndContributors">感谢这些帮助翻译此软件的人：
 Dong Cunrui (董存瑞)
+Aruvelut-123
 RichyZ99
 Jhonatan Lima (Heremitus)
 İskender ALPSAKARYA, Alp Eren GENÇOĞLU (Türkçe)
-Luis Duenas (Español)
```

Comment:
Why did you remove this credit?

Reply:
IDK，it's accident i think，let me put it back

Reply:
OK i add it back already

Reply:
IDK why i translating this will accidently delete some of these